cf_xarray/tests/test_helpers.py

```diff
@@ -18,8 +18,14 @@ def test_bounds_to_vertices():
     assert_array_equal(ds.lat.values + 1.25, lat_c.values[:-1])
 
     # 2D case, CF- order
-    lat_c = cfxr.bounds_to_vertices(mollwds.lat_bounds, bounds_dim="bounds")
-    assert_equal(mollwds.lat_vertices, lat_c)
+    lat_ccw = cfxr.bounds_to_vertices(
```

Comment:
should we keep the clockwise test case too?

Reply:
I think so, so now both cases are tested. Also because the new test only tests that cf_xarray is coherent with itself. The second one further down tests against verified bounds and vertices.